scripts/content-repo-preview/build.ts

```diff
@@ -101,14 +96,8 @@ async function main() {
   /** Build */
   execFileSync('npm', ['run', 'build'], { stdio: 'inherit' })
 
-  // Copy the .next folder to the top-level so the site can be deployed and we can take advantage of the cache
-  // We are using cp here because the Vercel build is still running in the context of the nested website-preview directory,
-  // and so it needs the .next folder available to complete the deployment
-  execFileSync('cp', ['-R', '.next', '../.next'], { stdio: 'inherit' })
-
   // Put node_modules into .next/cache so we can retrieve them on subsequent builds
-  // We are using mv here because it is more efficient than cp, and node_modules is a big chungus
-  execFileSync('mv', ['node_modules', '../.next/cache/node_modules'], {
```

Comment:
We can't use `mv` here unfortunately, as the deployment process depends on the node_modules being in the right spot 😅  Using cp is a bit slower, but doing this still provides a nice speedup for builds.